platform/linux-generic/odp_schedule_scalable.c

```diff
@@ -1791,31 +1807,31 @@ static int schedule_init_global(void)
 			    (ODP_SCHED_PRIO_NUM * MAXTHREADS - 1) *
 			    sizeof(sched_queue_t);
 		max_alloc = min_alloc;
-		sched_shm_pool = _odp_ishm_pool_create("sched_shm_pool",
-						       pool_size,
-						       min_alloc, max_alloc,
-						       _ODP_ISHM_SINGLE_VA);
-		if (sched_shm_pool == NULL) {
+		pool = _odp_ishm_pool_create("sched_shm_pool", pool_size,
```

Comment:
here will be also good to rename it to _odp_shed_shm_pool so in stats it will be visible that it's internal odp allocation (platfrom), not odp application. Look up function also needs to be corrected.